flow-model-generator/modules/flow-model-generator-core/build.gradle

```diff
@@ -52,7 +52,7 @@ dependencies {
         transitive = false
     }
 
-    implementation "io.ballerina.openapi:core:2.1.1-20240926-171100-1f88ade"
+    implementation "io.ballerina.openapi:core:2.2.0-20241205-142900-95f0235"
```

Comment:
Shall we extract the version to `gradle.properties`?

Reply:
Fixed